### DISCUSSION TITLE
how to us System.IO. to manage file

### DISCUSSION THREAD

**Discussion by DJDQfff** [unoplatform/uno#13438] [General]
I have the path of target file , now I want to delete it, I tried these two ways to do it:
 * using filepicker to get the StorageFIle ,then delete it. but for multiple files , it is impossible to pick the folder or file each time.
 * System.IO.File.Delete(path), but the path is like content://com.android.externalstorage.documents/document/primary%3A.lrc  ,  but the path cannot be recognized . 

I used to use StorageAccessPermission to manage the file  in WinUI , but it is not implemented in UNO.

**Comment by jeromelaban** [comment]
Thanks for the report. Each platform has its own limitations, are you trying to work with android only? If you can attach a full sample of what you're trying to do we may be able to help.
